src/core/actions/categories.js

```diff
@@ -1,13 +1,15 @@
 import {
   CATEGORIES_FETCH,
   CATEGORIES_LOAD,
-  CATEGORIES_FAIL,
 } from 'core/constants';
 
-export function categoriesFetch() {
+export function categoriesFetch({ errorHandlerId } = {}) {
+  if (!errorHandlerId) {
```

Comment:
Is that still a goal though? I thought the outcome of our discussions in San Francisco was that Flow was useful for propTypes and maybe state/API annotations, but not every single function.